### PR TITLE
ThrowingRequest

### PR DESCRIPTION
**Pull request by Misaka-ZeroTwo**
Remove IsAllowed to "fix" #1746

### CODE REVIEW

**Review by louis1706 — APPROVED** [review]

File: Exiled.Events/EventArgs/Player/ThrowingRequestEventArgs.cs [hunk 1]
```diff
@@ -7,17 +7,18 @@
...
     using Exiled.API.Features.Items;
     using Exiled.Events.EventArgs.Interfaces;
-
     using InventorySystem.Items.ThrowableProjectiles;
 
     /// <summary>
     /// Contains all information before receving a throwing request.
     /// </summary>
-    public class ThrowingRequestEventArgs : IPlayerEvent, IDeniableEvent
+    public class ThrowingRequestEventArgs : IPlayerEvent
```
Comment (louis1706) [inline Exiled.Events/EventArgs/Player/ThrowingRequestEventArgs.cs:21]: this change should not be done you need to make a TODO

**Review by louis1706 — CHANGES_REQUESTED** [review]
Oups

**Review by louis1706 — CHANGES_REQUESTED** [review]
Oups

**Review by IRacle1 — APPROVED** [review]

**Comment by Misaka-ZeroTwo** [comment]
Do not merge, will make it a breaking change since it’s exiled 7

**Review by louis1706 — APPROVED** [review]

**Review by louis1706 — APPROVED** [review]

**Pull request merged**